concordium-consensus/src/Concordium/KonsensusV1/LeaderElection.hs

```diff
@@ -147,3 +156,29 @@ updateSeedStateForEpoch newBakers epochDuration ss =
         }
   where
     newNonce = nonceForNewEpoch newBakers ss
+
+-- | Compute the missed rounds for each validator. Starts from the given parent
+--   block up to (not including) the given final round.
+computeMissedRounds ::
+    -- | Optional timeout certificate for the round of the parent block.
+    Option TimeoutCertificate ->
+    -- | Validators participating in the current epoch.
+    FullBakers ->
+    -- | Leadership election nonce for the current epoch.
+    LeadershipElectionNonce ->
+    -- | Pointer to the parent block.
+    BlockPointer mpv ->
+    -- | The current round.
+    Round ->
```

Comment:
This is not really needed, because the current round will be `tcRound tc + 1` if the timeout certificate is present.

Reply:
Can't the `tcRound` point to some earlier round?

Reply:
I removed the superfluous parent round parameter from `computeMissedRounds`.